### ISSUE TITLE
Can I use this with Mojave installation on HP-Z620

### ISSUE THREAD

**Issue by bertilli** [d4vinder/HP-Z620-Hackintosh-macOS-Catalina#4]
Hi.
Your work of installing Catalina on Z620 is amazing. No issues and very smooth installation. Perfect. Job well done. Thank you so very much.
In my case I need Mojave installed on my Z620. I have done tons of search and did a few tries. I could not get it running. Very frustrated. Do you think you can create the same installation for Mojave as well?
By the way I have EVGA GeForce GT 740 graphics card.
Thanks.
Chet

**Comment by d4vinder** [comment]
I have not tried this with Mojave unfortunately and I do not have any immediate plans to roll back to it either. Sorry. 

Your 740 GPU should be supported natively in Mojave.

**Comment by bertilli** [comment]
Thank you for getting back to me so soon. I get to the install macOS but I can not get passed "Installing macOS Mojave on this Mac requires that all graphics cards have metal support and that FileVault is disabled" Any ideas What I can do to pass it?
[config.plist.zip](https://github.com/d4vinder/HP-Z620-Hackintosh-macOS-Catalina/files/4740795/config.plist.zip)
Your help greatly be appreciated.

**Comment by d4vinder** [comment]
Sounds like your GPU isn't supported by the current configuration of the EFI. Have a look to omit any references to NVIDIA or ATI and remove them. Please note this an issues thread for the guide attached to this repo for Catalina. I cannot support your Mojave build further.

**Comment by bertilli** [comment]
I figured it out.. I was able to install and boot in to Mojave very successfully. I had to use MacPro 6,1 instead. That fixed the GPU problem. What I am having a problem with is USB 3. It can see and read the data but I cannot copy in or out any data bigger than 1 MB. I am not sure the sleep wake issue is caused by the USB3 or not but I am also having a great deal of issues with the sleep wake. it goes to sleep then auto wakes up in a bad state which runs the fans too high..  Thank you for the answer by the way.
